src/custom/api/gnosisProtocol/index.ts

```diff
@@ -10,7 +10,6 @@ const useMock = process.env.REACT_APP_MOCK === 'true'
 // without manually naming the exports
 export const {
   // functions that have a mock
-  getAppDataDoc,
   uploadAppDataDoc,
```

Comment:
@alfetopito does this `uploadAppDataDoc` still make any sense? or should I remove it on this PR to wrap up all the Affiliate related changes?

Reply:
You are right, it no longer applies as it'll be uploaded directly to IPFS.
Please remove it